src/identity/hcs/did/hcs-did-topic-listener.ts

```diff
@@ -31,19 +31,27 @@ export class HcsDidTopicListener extends MessageListener<HcsDidMessage> {
 
     protected override isMessageValid(envelope: MessageEnvelope<HcsDidMessage>, response: TopicMessage): boolean {
         try {
-            const msgDecrypter = !!this.decrypter ? HcsDidMessage.getDecrypter(this.decrypter) : null;
+            // const msgDecrypter = !!this.decrypter ? HcsDidMessage.getDecrypter(this.decrypter) : null;
+            /**
+             * TODO: Looks like we no longer encrypt messages
```

Comment:
we should no longer encrypt messages, 
going forward our messages will be only events, and must be publicly readable ( unencrypted )